styles/components/_steps.scss

```diff
@@ -0,0 +1,47 @@
+.steps {
+  padding: 8px 15px;
+  list-style: none;
+  color: $brand-gray;
+  font-size: $small;
+
+  li {
+    position: relative;
+    display: inline-block;
+    margin: 0 1em;
+    width: 2em;
+    height: 2em;
+    padding: 2px 10px;
+    text-align: center;
+    background: $brand-gray-lt;
+    text-decoration: none;
+    border-radius: 1em;
+    border: $border-radius-base solid $brand-gray;
+  }
+
+  li > a { color: $brand-gray-dk; }
+
+  li::before{
+    content: '';
+    position: absolute;
+    top: 0.9em;
+    left: -4em;
+    width: 4em;
+    height: 0.2em;
+    background: $brand-gray;
+    z-index: -1;
```

Comment:
If you're adding a `z-index` here, make sure to add a `z-index` too for its sibling elements (which are used in combination). It's `a` in this case, I think?